src/cpp/shared/combat/army.cpp

```diff
@@ -614,6 +650,293 @@ void army::Walk(signed int dir, int last, int notFirst) {
   }
 }
 
+int army::FindPath(int knownHex, int targHex, int speed, int flying, int flag) {
+  int res;
+  std::vector<int> obstacleHexes;
+
+  if(!IsAICombatTurn() && CreatureHasAttribute(this->creatureIdx, JUMPER) && gIronfistExtra.combat.stack.abilityCounter[this]) {
+    for (int i = 0; i < 117; i++) {
```

Comment:
Please replace all relevant uses of 117 in the cpp code with NUM_HEXES or something.